src/app/containers/Metadata/index.jsx

```diff
@@ -102,6 +103,9 @@ const MetadataContainer = ({
   const metaImage = image || defaultImage;
   const metaImageAltText = imageAltText || defaultImageAltText;
 
+  // For Ukrainian pages with Russian content we want to use the `ru_UA` locale
+  const contentLang = metaLocale || locale;
```

Comment:
I believe we can just use the `locale` of `ru_UA` instead of creating a new metaLocale field.

Reply:
Initially, I was worried that the locale value might have been being used in other parts of our application (like timestamp related stuff) but from what I can see it is not. It is only used in the metadata container and radio schedule container.

Reply:
I suppose locale in radio schedules could also be updated to use `dateLocale` 🤔